text/000x-lowering-deposits-assethub.md

```diff
@@ -0,0 +1,197 @@
+| attributeDepositBase      | 0.00666666666 KSM                | 0.000666666666 KSM        |
+
+
+## Discussion of Other Proposals
+Several innovative proposals have been considered to enhance the network's adaptability and manage deposit requirements more effectively:
+
+### Enhanced Weak Governance Origin Model
+The concept of a weak governance origin, controlled by a consortium like the System Collective, has been proposed. This model would allow for dynamic adjustments of NFT deposit requirements in response to market conditions, adhering to storage deposit norms. 
+
+**Enhancements and Concerns:**
+- **Responsiveness**: To address concerns about delayed responses, the model could incorporate automated triggers based on predefined market indicators, ensuring timely adjustments.
+- **Stability vs. Flexibility**: Balancing stability with the need for flexibility is challenging. To mitigate the issue of frequent changes in DOT-based deposits, a mechanism for gradual and predictable adjustments could be introduced.
+- **Scalability**: The model's scalability is a concern, given the numerous deposits across the system. A more centralized approach to deposit management might be needed to avoid constant, decentralized adjustments.
+
+### Function-Based Pricing Model
+Another proposal is to use a mathematical function to regulate deposit prices, initially allowing low prices to encourage participation, followed by a gradual increase to prevent network bloat.
+
+**Refinements:**
+- **Choice of Function**: A logarithmic or sigmoid function is favored over an exponential one, as these functions increase prices at a rate that encourages participation while preventing prohibitive costs.
+- **Adjustment of Constants**: To finely tune the pricing rise, one of the function's constants could correlate with the total number of NFTs on AssetHub. This would align the deposit requirements with the actual usage and growth of the network.
+
+### Linking Deposit to USD(x) Value
+This approach suggests pegging the deposit value to a stable currency like the USD, introducing predictability and stability for network users.
+
+**Considerations and Challenges:**
+- **Market Dynamics**: One perspective is that fluctuations in native currency value naturally balance user participation and pricing, deterring network spam while encouraging higher-value collections. Conversely, there's an argument for allowing broader participation if the DOT/KSM value increases.
+- **Complexity and Risks**: Implementing a USD-based pricing system could add complexity and potential risks. The implementation needs to be carefully designed to avoid unintended consequences, such as excessive reliance on external financial systems or currencies.
+
+Each of these proposals offers unique advantages and challenges. The optimal approach may involve a combination of these ideas, carefully adjusted to address the specific needs and dynamics of the Polkadot and Kusama networks.
```

Comment:
I think we can probably do a combination of these two. It'd require some implementation, so switching to `deposit()` would be phase one, and then we could do something like:

- Function 1: Deposit (in DOT) as a function of number of collections.
- Function 2: Fixed USD(x) cost as a ceiling.
- Actual deposit: `min(F1, F2)`

Reply:
I'm curious about the motivation behind increasing the deposit based on the number of collections. If the proposed floor deposits are too low to prevent spam, in theory, the function will adjust the deposit value to a more reasonable one and continue to grow, potentially raising a barrier for creators again.

This leads me to think that the primary challenge is to find an optimal deposit value, which depends on the DOT/KSM price and our current technology state (as mentioned by @burdges [here](https://github.com/polkadot-fellows/RFCs/pull/45#issuecomment-1798950680)). If there is no optimal value, we will need to explore technical solutions.

If what I've said withstands criticism, we could have a scale factor parameter that can be changed through a referendum and deposits could also be linked to the USD price. Additionally, we should explore and learn how to estimate such an optimal deposit value.